rsshfs

```diff
@@ -20,6 +20,18 @@ then
     exit 1;
 fi
 
+# get optional identity file
+unset identity
+if [ "$1" = "-i" ]; then
+    if [ -f "$2" ]; then
+        identity="-i $2"
```

Comment:
Hello,

I know/noticed this patch does not comply with your coding
standards, and I'm aware of the space problem,
The .ssh/config is not an option.
I agree that a generic way to pass ssh options through rsshfs
command line would be nice.
So when I have time, I'll check into this.


On lun., avril 18, 2016 at 04:37:53 -0700, ®om wrote:
> > @@ -20,6 +20,18 @@ then
> >      exit 1;
> >  fi
> >  
> > +# get optional identity file
> > +unset identity
> > +if [ "$1" = "-i" ]; then
> > +    if [ -f "$2" ]; then
> > +        identity="-i $2"
> 
> Does not work if `$2` contain spaces.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/rom1v/rsshfs/pull/1/files/2b478f205ce9127c72259d826f7d72de95986d3b#r60043386